apps/web/layouts/main.tsx

```diff
@@ -199,9 +201,28 @@ const Layout: NextComponentType<
     },
   ]
 
-  const alertBannerId = `alert-${stringHash(
-    JSON.stringify(alertBannerContent),
-  )}`
+  const [alertBanners, setAlertBanners] = useState([])
+
+  useEffect(() => {
+    setAlertBanners(
+      [
+        {
+          bannerId: `alert-${stringHash(
+            JSON.stringify(alertBannerContent ?? {}),
+          )}`,
+          ...alertBannerContent,
+        },
+        {
+          bannerId: `organization-alert-${stringHash(
+            JSON.stringify(organizationAlertBannerContent ?? {}),
+          )}`,
+          ...organizationAlertBannerContent,
+        },
+      ].filter(
+        (banner) => !Cookies.get(banner.bannerId) && banner?.showAlertBanner,
+      ),
+    )
+  }, [alertBannerContent, organizationAlertBannerContent])
```

Comment:
Just wondering, is there any particular reason the banners are stored in state? looks like the content is never changing, and the bannerId is stored in cookie

```suggestion
  const alertBanners = [
    {
      bannerId: `alert-${stringHash(
        JSON.stringify(alertBannerContent ?? {}),
      )}`,
      ...alertBannerContent,
    },
    {
      bannerId: `organization-alert-${stringHash(
        JSON.stringify(organizationAlertBannerContent ?? {}),
      )}`,
      ...organizationAlertBannerContent,
    },
  ].filter(
    (banner) => !Cookies.get(banner.bannerId) && banner?.showAlertBanner,
  )
```

Reply:
I placed the alert banners in state for two reasons. Number one, since now we don't have to access the cookie on every re-render. Number two, because after I dismissed the alert banners and hit refresh I still saw the background color of them even though they weren't supposed to be rendered and somehow adding them to the state fixed that issue:
![image](https://user-images.githubusercontent.com/43557895/155497488-d82b8bf8-573e-4f98-a0c8-aa65f7c6e520.png)

Reply:
That's weird, I don't see why the content would not be populated in the banner if this is a cookie issue. There might be something off with the AlertBanner component, but if this works it should be fine for now 🙃